torch/_subclasses/fake_impls.py

```diff
@@ -411,7 +413,9 @@ def repeat_interleave_tensor(fake_mode, func, repeats, output_size=None):
 @register_op_impl(torch.ops.aten.item.default)
 @register_op_impl(torch.ops.aten._local_scalar_dense.default)
 def local_scalar_dense(fake_mode, func, arg):
-    if (r := arg.item_memo) is not None:
+    if (
```

Comment:
is the item memo associated with the fake_tensor? 

Disabling item_memo may not be what we want  e.g. users might be doing something like:
```python
def body_fn(c):
  for _ in range(10):
    c_val = c.item()
    ...
```
In this case, we don't want to create an unbacked symint every iteration since they're essentially the same .item().